windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -100,39 +134,43 @@ if "%OS%" == "32-BIT" (
 
 
 :_choice
-set _ok=
-set /p _ok= Choose an option or press Enter to close: ||goto:eof
-if "%OS%" == "32-BIT" IF "%_ok%" == "1" SET CHOICE=32Bit&GOTO :32Bit
-if "%OS%" == "64-BIT" IF "%_ok%" == "1" SET CHOICE=64Bit&GOTO :64Bit
+SET _ok=
+SET /p _ok= Choose an option or press Enter to close: ||goto:eof
+IF "%OS%" == "32-BIT" IF "%_ok%" == "1" SET CHOICE=32Bit&GOTO :32Bit
+IF "%OS%" == "64-BIT" IF "%_ok%" == "1" SET CHOICE=64Bit&GOTO :64Bit
 IF "%_ok%" == "2" SET CHOICE=InstallBot&GOTO :InstallBot

```

Comment:
Do we need to offer a choice between architectures? Cant we just install the correct one and log that?